src/herder/TransactionQueue.cpp

```diff
@@ -156,12 +167,15 @@ TransactionQueue::extract(TransactionFramePtr const& tx, bool keepBacklog)
         // remove everything passed tx
         txRemoveEnd = std::end(txs);
     }
-    auto removedFeeBid =
-        std::accumulate(txIt, txRemoveEnd, int64_t{0},
-                        [](int64_t fee, TransactionFramePtr const& tx) {
-                            return fee + tx->getFeeBid();
-                        });
-    accIt->second.mTotalFees -= removedFeeBid;
+    using feeOps = std::pair<int64_t, size_t>;
+    auto removedFeeOps = std::accumulate(
```

Comment:
Style: this isn't idiomatic. No need to be so compact, just call `std::accumulate` twice to compute the different values you want.

Reply:
actually code is dumb, with a simple loop it's much simpler to write